### ISSUE TITLE
Pupil Player using 70-80&% processor while idling [Mac Os]

### ISSUE THREAD

**Issue by VisionResearchBlog** [pupil-labs/pupil#1347]
Late 2013 MacBook Pro Intel i7 - Mac OS 10.14 Mojave - 

Running pupil player 1.8.26 and not loading anything, just running in background is using 70-80% processor resources according to activity monitor with fan kicking in

~BT

**Comment by papr** [comment]
Are you referring to the gray window where you can drop recordings on?

**Comment by VisionResearchBlog** [comment]
Yes, I start the app and then do not interact with it at all - I noticed this today as I started working in other apps program to answer emails/go on the web - I forgot I had started pupil player and noticed the fan on my laptop was running - In activity monitor the player app was reporting 70-80% use.

**Comment by VisionResearchBlog** [comment]
Update, this also holds true when tracking data is loaded and the video is paused and the software is otherwise not being interacted with

**Comment by VisionResearchBlog** [comment]
Update: Using Pupil Player 1.9, same problem -  as stated before I start pupil player & let it idle in the background (I don’t even load anything just start the app and then leave it alone) it starts to use 70-80%+ CPU resources and kicks in my CPU fan. What I noticed though is that this appears on how I let the app idle – if I minimize to the dock it uses too much CPU, if I cover the window with another window (say my email client) it also uses too much CPU, but if the app is idling and is not covered by another app window or minimized it only uses about 3% CPU.  Not sure why it does this but at least this is a little more info over what I noticed before.

**Comment by papr** [comment]
I am able to reproduce the issue if I switch to a different virtual space without the player window. I think the issue is related to glfw but I cannot say for sure. We will look into it.
